src/ManageCourses.Api/Mapping/EnrichmentConverter.cs

```diff
@@ -64,11 +64,11 @@ public UcasProviderEnrichmentGetModel Convert(ProviderEnrichment source)
             var enrichmentModel = source.JsonData != null ? JsonConvert.DeserializeObject<ProviderEnrichmentModel>(source.JsonData, _jsonSerializerSettings) : null;
 
             enrichmentToReturn.EnrichmentModel = enrichmentModel;
-            enrichmentToReturn.CreatedTimestampUtc = source.CreatedTimestampUtc;
-            enrichmentToReturn.UpdatedTimestampUtc = source.UpdatedTimestampUtc;
+            enrichmentToReturn.CreatedTimestampUtc = source.CreatedAt;
+            enrichmentToReturn.UpdatedTimestampUtc = source.UpdatedAt;
             enrichmentToReturn.CreatedByUserId = source.CreatedByUser.Id;
             enrichmentToReturn.UpdatedByUserId = source.UpdatedByUser.Id;
-            enrichmentToReturn.LastPublishedTimestampUtc = source.LastPublishedTimestampUtc;
+            enrichmentToReturn.LastPublishedTimestampUtc = source.LastPublishedAt;
```

Comment:
What about just `published_at`?

Reply:
Just had a look with @timabell and this attribute is a bit quirky, so we'll leave it as-is and consider refactoring at a later time.